lib/components/app/desktop-nav.tsx

```diff
@@ -46,9 +46,29 @@ const DesktopNav = ({
   popupTarget,
   setPopupContent
 }: Props) => {
-  const { branding, persistence, title = DEFAULT_APP_TITLE } = otpConfig
+  const {
+    branding,
+    persistence,
+    title = DEFAULT_APP_TITLE,
+    brandClickable
+  } = otpConfig
   const showLogin = Boolean(getAuth0Config(persistence))
 
+  const BrandingElement = brandClickable ? 'a' : 'div'
+
+  const commonStyles = { marginLeft: 50 }
+  const brandingProps = brandClickable
+    ? {
+        href: '/#/',
```

Comment:
There is an existing issue with the router or the code that causes some parts of the UI to not pick up all the changes, so this will not be the same as "Start Over". For instance, after you plan an itinerary and click the logo, most of the map will clear, however the locations in the location fields and location markers will remain.

To trigger a page reload, the URL should not include the hash. Reuse [this method to compute the restart URL](https://github.com/opentripplanner/otp-react-redux/blob/dev/lib/components/util/start-over.ts) that is used by "Start Over" button in itinerary and app menu. It also accommodates clients that have the UI in a subpath of their site.